src/Symfony/Component/AssetMapper/Command/ImportMapOutdatedCommand.php

```diff
@@ -76,6 +76,8 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $packagesUpdateInfos = $this->updateChecker->getAvailableUpdates($packages);
         $packagesUpdateInfos = array_filter($packagesUpdateInfos, fn ($packageUpdateInfo) => $packageUpdateInfo->hasUpdate());
         if (0 === \count($packagesUpdateInfos)) {
+            $io->writeln('Everything is up to date');
```

Comment:
I think we should respect the format option here.

Reply:
OK. But what would be the JSON version? Something like:
```json
{"message": "Everything is up to date"}
```

Reply:
Maybe this should be considered as a reporting message rather than an output message.

For JSON, the only sensible output would be an empty array (returning a different JSON schema would break any scripting)

Reply:
OK, returning `[]` now :-)